docs/content/basic/authenticating-oauth.md

```diff
@@ -4,256 +4,703 @@ lang: en
+All of the additional underlying details around authentications can be found at
+[api.slack.com/authentication/oauth-v2][oauth-v2]!
 
----
+## Configuring the application
+
+To set your Slack app up for distribution, you will need to enable Bolt OAuth
+and store installation information securely. Bolt supports OAuth by using
+[`@slack/oauth`][oauth-node] to handle most of the work; this includes setting
+up OAuth routes, state verification, and passing your app an installation object
+which you must store.
+
+### App options
+
+The following `App` options are required for OAuth installations:
 
-## Development and Testing
+- `clientId`: unique identifier of the application client.
+- `clientSecret`: secret value to confirm the client ID.
+- `stateSecret`: secret value used for [state verificiation][verification] of
+  authentication requests.
+- `scopes`: permissions requested for the `bot` user during installation.
+  [Explore scopes][scopes].
+- `installationStore`: handlers that store, fetch, and delete installation
+  information.
 
-We've provided a default implementation of the `installationStore`  `FileInstallationStore` which you can use during app development and testing.
+#### Development and testing
+
+Here we've provided a default implementation of the `installationStore` with
+[`FileInstallationStore`][store-file-installation] which can be useful when
+developing and testing your app:
 
 ```javascript
-const { App } = require('@slack/bolt');
-const { FileInstallationStore } = require('@slack/oauth');
+const { App } = require("@slack/bolt");
+const { FileInstallationStore } = require("@slack/oauth");
 const app = new App({
   signingSecret: process.env.SLACK_SIGNING_SECRET,
   clientId: process.env.SLACK_CLIENT_ID,
   clientSecret: process.env.SLACK_CLIENT_SECRET,
-  stateSecret: 'my-state-secret',
-  scopes: ['channels:history', 'chat:write', 'commands'],
+  stateSecret: "my-state-secret",
+  scopes: ["channels:history", "chat:write", "commands"],
   installationStore: new FileInstallationStore(),
 });
 ```
-:::warning 
 
-This is **_not_** recommended for use in production - you should implement your own production store. Please see the example code to the right and [our other examples](https://github.com/slackapi/bolt-js/tree/main/examples/oauth).
+:::warning
+
+This is **not** recommended for use in production - you should
+[implement your own installation store](#installation-store). Please continue
+reading or inspect [our example apps][examples].
 
 :::
 
+### Installer options
+
+We provide several options for customizing default OAuth using the
+`installerOptions` object, which can be passed in during the initialization of
+`App`. You can override these common options and
+[find others here][install-provider-options]:
+
+- `authVersion`: Settings for either new Slack Apps (`v2`) or Classic Slack Apps
+  (`v1`). Default: `v2`.
+- `directInstall`: Skip the [default installation page][installation] at
```

Comment:
skip **rendering (or displaying)** the page?

Reply:
Added "rendering" here but also a few more words about the alternative 🪙 🤔  Still open to more updates or removing a bit!